server/odc-migrate/src/main/resources/migrate/common/V_4_3_1_4__add_logical_database.sql

```diff
@@ -0,0 +1,57 @@
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
+alter table connect_database add column `type` varchar(32)  NOT NULL DEFAULT 'PHYSICAL' COMMENT 'optional value: PHYSICAL, LOGICAL';
+alter table connect_database add column `alias` varchar(256) DEFAULT NULL COMMENT 'alias name for database';
+alter table `connect_database` modify `last_sync_time` datetime default null comment 'last synchronizing time';
+
+
+create table if not exists `connect_logical_db_meta`(
```

Comment:
how about merge into connect_database, it already contains unique id now

Reply:
Nice suggestion~ I updated it.